scripts/maxmind-db-downloader.js

```diff
@@ -0,0 +1,126 @@
+    var targetDirPath = path.join(__dirname, '..', targetDirName);
+    // create db folder
+    mkdirp.sync(targetDirPath);
+    // log.info only if not testing
+    // log.error always
+    if (process.env.NODE_ENV !== 'test') {
+      log.info('Download folder is ' + targetDirPath);
+    }
+    return targetDirPath;
+  };
+
+  this.setupDownloadList = function (sourceFilePath, targetDirPath) {
+    sourceFilePath = sourceFilePath || DEFAULTS.DEFAULT_SOURCE_FILE_NAME;
+    targetDirPath = targetDirPath || DEFAULTS.DEFAULT_TARGET_DIR_PATH;
+    // import the list of files to download
+    var sources = require(sourceFilePath);
+    var remainingDownloads = [];
+
+    // push each file-load-function onto the remainingDownloads queue
+    for (var source in sources) {
+      var url = sources[source];
+      if (process.env.NODE_ENV !== 'test') {
+        log.info('Adding ' + url);
+      }
+      // get the file name without the extension
+      var targetFileName = path.parse(source).name;
+      var targetFilePath = path.join(targetDirPath, targetFileName);
+      remainingDownloads.push(this.download(url, targetFilePath));
+      if (process.env.NODE_ENV !== 'test') {
+        log.info('Setting ' + targetFilePath + ' as target file');
+      }
+    }
+    return remainingDownloads;
+  };
+
+  this.download = function (url, targetFilePath) {
+    // closure to separate multiple file-download
+    return function downloadFunctor(callback) {
+      var stream = request(url);
+      stream.pipe(zlib.createGunzip()).pipe(fs.createWriteStream(targetFilePath)).on('finish', function (err) {
+        // forces overwrite, even if file exists already
+        if (err) {
+          log.error(err);
+        } else {
+          // extraction is complete
+          if (process.env.NODE_ENV !== 'test') {
+            log.info('unzip complete');
+          }
+          callback();
+        }
+      });
+    };
+  };
+
+  this.startDownload = function (remainingDownloads) {
+    async.parallel(remainingDownloads, function (err) {
+      if (err) {
+        log.error(err);
+      } else {
+        if (process.env.NODE_ENV !== 'test') {
+          log.info('Downloads complete');
+        }
+      }
+    });
+    if (process.env.NODE_ENV !== 'test') {
+      log.info('Last Update: ', new Date());
+    }
+  };
+
+  this.setupAutoUpdate = function (cronTiming, remainingDownloads, timeZone) {
+    cronTiming = cronTiming || DEFAULTS.DEFAULT_CRON_TIMING;
+    timeZone = timeZone || DEFAULTS.DEFAULT_TIMEZONE;
+    var self = this;
+    // by default run periodically on Wednesday at 01:30:30.
+    new CronJob(cronTiming, function() { // eslint-disable-line no-new
+      self.startDownload(remainingDownloads);
+    }, null, true, timeZone);
+    if (process.env.NODE_ENV !== 'test') {
+      log.info('Set up auto update with cronTiming: ' + cronTiming);
+    }
+  };
+};
+
+if (require.main === module) {
```

Comment:
I personally would move most of the downloader code into a module in `src` [or `lib`](https://github.com/mozilla/fxa-geodb/pull/1/files/75c02a076917d3db24e729d3c31dbf5652208739#r70795185), which would export MaxmindDbDownloder. The only thing that'd remain in this module are lines 111-124. The separation of concerns would make each easier to reason about.